.github/workflows/ci.yml

```diff
@@ -194,6 +194,8 @@ jobs:
         env:
           GOTESTSUM_FORMAT: standard-verbose
           GOTESTSUM_JUNITFILE: unit-tests.xml
+          # We set this to prevent segmentation faults on macOS see - https://github.com/golang/go/issues/49138
+          MallocNanoZone: 0
```

Comment:
This issue has been fixed in Go 1.17 if I understand the linked issue correctly, we are already using Go 1.20 🙂

Reply:
I can't find the run now, but I was seeing SIGSEV errors, I've removed it, lets see what happens.